src/component-library/CheckCircleIcon/CheckCircleIcon.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import { Story, Meta } from '@storybook/react';
```

Comment:
I think it's unnecessary to create separate component for checkmark icon as we already have general `Icon` component in the component library that can be used.

Reply:
Done!